src/operators.jl

```diff
@@ -39,10 +39,6 @@
 (-){T<:GenericSOCExpr}(lhs::Number, rhs::T) = T(copy(rhs.norm),    -rhs.coeff, lhs-rhs.aff)
 (*){T<:GenericSOCExpr}(lhs::Number, rhs::T) = T(copy(rhs.norm), lhs*rhs.coeff, lhs*rhs.aff)
 
-# This is not well defined if variable types are different, but needed to avoid ambiguities
-(+)(lhs::GenericAffExpr, rhs::GenericAffExpr) = (+)(promote(lhs,rhs)...)
```

Comment:
Removing these definitions broke JuMPChance. I have:
```
Base.promote_rule(::Type{AffExpr},::Type{CCAffExpr}) = CCAffExpr
```
which saves a bunch of method definitions.